tests/test_addon_detail.py

```diff
@@ -111,6 +113,106 @@ def test_platform_incompatibility(selenium, base_url, variables):
     assert addon.button_state_disabled
 
 
+@pytest.mark.nondestructive
+def test_addon_with_stats_summary(selenium, base_url, variables):
+    extension = variables['addon_with_stats']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    # checks that a summary of users, reviews and star ratings are present
+    assert addon.stats.stats_users_count > 0
+    assert addon.stats.stats_reviews_count > 0
+    assert addon.stats.addon_star_rating_stats.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_addon_without_stats_summary(selenium, base_url, variables):
+    extension = variables['addon_without_stats']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert 'No Users' in addon.stats.no_user_stats
+    assert 'No Reviews' in addon.stats.no_reviews_stats
+    assert 'Not rated yet' in addon.stats.no_star_ratings
+
+
+@pytest.mark.nondestructive
+def test_stats_reviews_summary_click(selenium, base_url, variables):
+    extension = variables['addon_with_stats']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    stats_review_counts = addon.stats.stats_reviews_count
+    # clicks on reviews stats link to open all reviews page
+    addon.stats.stats_reviews_link.click()
```

Comment:
This click could return the ```Reviews``` object. Is there another use for ```stats_reviews_link``` outside of clicking and loading the Reviews page?